apel/db/records/htcondorce.py

```diff
@@ -0,0 +1,41 @@
+'''
+   Copyright 2012 Konrad Jopek
```

Comment:
I guess I am not author of this file :)

Reply:
I'll sort out all the notices before anything's merged to dev.